deploy/olm-catalog/integreatly-operator/integreatly-operator-2.0.0/integreatly-operator.v2.0.0.clusterserviceversion.yaml

```diff
@@ -234,6 +234,7 @@ spec:
           - monitoring.coreos.com
           resources:
           - prometheusrules
+          - servicemonitors
```

Comment:
Are we still on version 2.0.0 with these changes ? or is this something we change when we cut a release ?

Reply:
I believe this will be bumped in the next release, to 2.1.0

@philbrookes Can you confirm that this is the pattern we'll follow? e.g. A minor bump only on each new release?